webviz_subsurface/plugins/_surface_viewer_fmu.py

```diff
@@ -740,23 +741,34 @@ def save_surface(fns, statistic) -> io.BytesIO:
     surfaces = xtgeo.Surfaces(fns)
     if len(surfaces.surfaces) == 0:
         surface = xtgeo.RegularSurface()
-    elif statistic == "Mean":
-        surface = surfaces.apply(np.nanmean, axis=0)
-    elif statistic == "StdDev":
-        surface = surfaces.apply(np.nanstd, axis=0)
-    elif statistic == "Min":
-        surface = surfaces.apply(np.nanmin, axis=0)
-    elif statistic == "Max":
-        surface = surfaces.apply(np.nanmax, axis=0)
-    elif statistic == "P10":
-        surface = surfaces.apply(np.nanpercentile, 10, axis=0)
-    elif statistic == "P90":
-        surface = surfaces.apply(np.nanpercentile, 90, axis=0)
+    elif statistic in ["Mean", "StdDev", "Min", "Max", "P10", "P90"]:
+        # Suppress numpy warnings when surfaces have undefined z-values
```

Comment:
```
 # Suppress numpy warnings when surfaces have undefined z-values
```
is this something that should ideally be in `xtgeo`? Is it because some cells within the grid are inactive across all realizations? If so I guess that is a quite common/typical use case.

Reply:
Yes, that is the reason and I agree it is quite common.
The numpy operations that causes the errors are not in xtgeo though. 
We `apply` them. https://github.com/equinor/webviz-subsurface/pull/435/files#diff-3100bd9a3ac9669e2dab8a0115612cecR756

Reply:
Ah, ok :+1: